tests/foreman/ui/test_contentview_new.py

```diff
@@ -0,0 +1,44 @@
+"""Test class for Content View UI
+
+:Requirement: Contentview
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: ContentViews
+
+:Assignee: ltran
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+
+
+@pytest.mark.tier2
+def test_positive_create_cv(session):
+    """Able to create cv and search for it
+    :id: 1bad50d8-4909-47ef-8524-8c703c75069d
+
+    :CaseLevel: System
+    """
+    cv = gen_string('alpha')
+    with session:
```

Comment:
You could start using target_sat in this new module, and use ui_session instead of using this session fixture
```suggestion
    with target_sat.ui_session() as session:
```